src/algebra/homology/additive.lean

```diff
@@ -140,6 +140,27 @@ by tidy
 
 end category_theory
 
+namespace chain_complex
+
+variables {W : Type*} [category W] [preadditive W]
+variables {α : Type*} [add_right_cancel_semigroup α] [has_one α] [decidable_eq α]
+
+lemma map_of (F : V ⥤ W) [F.additive] (X : α → V) (d : Π n, X (n+1) ⟶ X n)
```

Comment:
I worry that this should have a longer name, for the sake of disambiguating.

Reply:
Would something like `map_homological_complex_obj_of` be ok?

Reply:
Eventually, it seems that `map_chain_complex_of` is a better name.